src/app.ts

```diff
@@ -48,6 +49,7 @@ export default (
   }));
   app.use(apiDocumentationLink(apiDocumentationPath));
   app.use(errorHandler());
+  app.use(article(articles));
```

Comment:
Did this not work as a router middleware in the end?

Reply:
Just experimented and it looks like router middleware aren't called if the route isn't found. Which makes sense.

Reply:
Just seen https://github.com/koajs/router/issues/44.